hw/ip/otbn/dv/smoke/smoke_test.S

```diff
@@ -0,0 +1,57 @@
+/* Copyright lowRISC contributors. */
+/* Licensed under the Apache License, Version 2.0, see LICENSE for details. */
+/* SPDX-License-Identifier: Apache-2.0 */
```

Comment:
Have you considered producing smoke_expected.txt from the magic comments in this file? That would avoid having to update two files at once on changes.

Reply:
I could do, just not sure we'd get much value out of it. It's not too hard to keep them in sync (though I should add a comment reminding people to keep them in sync though the CI will ensure that is done) and the simple solution (python script to regex for xN = ... = 0x.... in comments then choose the latest value for each xN to produce expected) feels a bit brittle.

Reply:
Hmm I do already have a reference to the expected file, think that will suffice

Reply:
Seems sensible!